### DISCUSSION TITLE
Demagnetized Zone

### DISCUSSION THREAD

**Discussion by Seikikai** [P3pp3rF1y/SophisticatedBackpacks#283] [Q&A] [answered]
There Are Mods/Datapacks Like [Create](https://www.curseforge.com/minecraft/mc-mods/create) Or [Eroding](https://www.curseforge.com/minecraft/mc-mods/eroding-stone-entities) That Works Using Systems That Involve To Drop Items And The Backpack Can Cause Malfunctions In Those Systems And Having To Enable/Disable The Backpack Magnetic Upgrade Wehn Entering That Zone Becomes Annoying.

My Suggestion Is To Add A Block Wich Prevents That Backpacks -Placed Or Equiped By An Entity- Above The Block From Picking Items In A Configurable Area.

**Comment by P3pp3rF1y** [comment, marked as answer]
you can use demagnetize mod for that. It marks the items with an attribute that's in general used to mark the items as not to be picked up by magnets and that sbp supports as well. Demagnetize basically gives you the block you're asking for here.
